app/assets/stylesheets/style.scss

```diff
@@ -209,3 +175,20 @@ button.link-button {
     margin-top: $gutter-half;
   }
 }
+.job-item {
+  p {
+    margin-bottom: 0.4em;
```

Comment:
there is not. this variable `p` is scoped only for job-item as everything there is like a house of cards. you change this variable to `0.3em` and everything falls apart.